test/bin/download_images.sh

```diff
@@ -71,5 +71,7 @@ title "Updating ostree references in ${IMAGEDIR}/repo"
 cd "${IMAGEDIR}"
 ostree refs --repo=repo --force \
        --create "rhel-9.2-microshift-failing" "rhel-9.2-microshift-source"
+ostree refs --repo=repo --force \
```

Comment:
This is a bit simpler than managing the version number in the image reference within the scenario script. Should we update https://github.com/openshift/microshift/blob/main/test/scenarios/upgrade-minor-version-healthy.sh#L9 as well?

Reply:
I just noticed that `download_images.sh` does not get `$PREVIOUS_MINOR_VERSION`.
I'm not sure but I think it worked when using `test/bin/rebuild_source_images.sh`.

I'll delete this line and follow the way from file you linked